SECURITY_RELEASE_PROCESS.md

```diff
@@ -0,0 +1,211 @@
+# Security Release Process
+
+Envoy is a large growing community of volunteers, users, and vendors. The Envoy community has adopted this security disclosures and response policy to ensure we responsibly handle critical issues.
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal communication and external disclosure but will need help from relevant developers and release managers to successfully run this process.
+
+The initial Product Security Team will consist of all [maintainers](OWNERS.md) in the private [envoy-security](https://groups.google.com/forum/#!forum/envoy-security) list.
+In the future we may decide to have a subset of maintainers work on security response.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+The Envoy community asks that all suspected vulnerabilities be privately and responsibly disclosed via the [reporting policy](README.md#reporting-security-vulnerabilities).
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY email [envoy-security](https://groups.google.com/forum/#!forum/envoy-security) to inform the Product Security Team (PST) about the vulnerability so they may start the patch, release, and communication process.
+
+If possible the PST will ask the person making the public report if the issue can be handled via a private disclosure process. If the reporter denies the PST will move swiftly with the fix and release process. In extreme cases you can ask GitHub to delete the issue but this generally isn't necessary and is unlikely to make a public disclosure less damaging.
+
+## Patch, Release, and Public Communication
+
+For each vulnerability a member of the PST will volunteer to lead coordination
+with the Fix Team, Release Managers and is responsible for sending disclosure
```

Comment:
I got where the fix team comes from but do we have documentation on release managers, how you get to be one, etc?

Reply:
We don't have them and I don't think we will for a while. I will refactor the text to be a bit closer to our current situation.